accounts/views.py

```diff
@@ -20,7 +26,49 @@ def post(self, request, format='json'):
         serializer = UserSerializer(data=request.data)
         if serializer.is_valid():
             user = serializer.save()
+            # TODO temporary overloading is_active field as a mean to confirm the email.
```

Comment:
Can you create a Asana ticket to track the TODO? We should follow this practice for all TODOs.

Reply:
I'm currently debating with myself whether the current implementation is also a long term solution. Sure I'll add a new task to the backlog.